tests/Unleash.Tests/Integration/ClientSpecificationTests.cs

```diff
@@ -38,7 +38,7 @@ static TestFactory()
 
             using (var client = new HttpClient())
             {
-                var csTestsVersion = "v4.3.3";
+                var csTestsVersion = "v4.5.1";
```

Comment:
4.5.2 is out! Adds a test case, which is relevant to dependent features!

```suggestion
                var csTestsVersion = "v4.5.2";
```